iosApp/iosApp/Home/SongCell.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  SongCell.swift
+//  iosApp
+//
+//  Created by Marty Ulrich on 1/9/22.
+//  Copyright © 2022 orgName. All rights reserved.
+//
+
+import SwiftUI
+
+struct SongCell: View {
+	struct Song: Identifiable {
+		let image: UIImage
+		let title: String
+		let artist: String
+		let isNFT: Bool
+		let songID: String
+		var id: ObjectIdentifier { title.objectIdentifier }
+	}
+	
+	var data: Song
+	
+	var body: some View {
+		VStack {
+			ZStack(alignment: .top) {
+				CircleImage(image: data.image, size: 60)
+				if data.isNFT {
+					nftLabel
+				}
+			}
+			Text(data.title)
+				.foregroundColor(.white)
+				.font(.caption3)
+			Text(data.artist)
+				.foregroundColor(.orange)
+				.font(.caption4)
+		}
+		.fixedSize()
+		.frame(width: 80, height: 150)
+	}
+	
+	private var nftLabel: some View {
```

Comment:
I think more than just songs can have NFT badges if you want to break this out into a more shareable place.  I am looking at designs that have artist cells with an NFT badge.

Reply:
put this in next pr